app/models/mixins/child_network_manager_mixin.rb

```diff
@@ -0,0 +1,46 @@
+module ChildNetworkManagerMixin
+  extend ActiveSupport::Concern
+
+  include BelongsToParentManagerMixin
+
+  PROVIDER_NAME = "Network Manager".freeze
+
+  included do
+    has_many :availability_zones,            -> { where.not(:ems_id => nil) }, :primary_key => :parent_ems_id, :foreign_key => :ems_id
+    has_many :flavors,                       -> { where.not(:ems_id => nil) }, :primary_key => :parent_ems_id, :foreign_key => :ems_id
```

Comment:
Do keep in mind that `ExtManagementSystem.includes(:flavors)` will return the flavors via the definition in ext management system and not the ones defined here